examples/web/src/features/resume/browser/styles.css

```
@@ -39,10 +45,21 @@
   --space-3xl: 48px;
   --space-4xl: 64px;
   --ease-out: cubic-bezier(0.23, 1, 0.32, 1);
+  min-width: 320px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)